Packs/FeedReversingLabsRansomwareAndRelatedToolsApp/pack_metadata.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "ReversingLabs Ransomware and Related Tools Feed",
+    "description": "A timely and curated threat intel list containing recent indicators extracted from ransomware and the tools used to deploy ransomware which are suitable for threat hunting or deployment to security controls.",
+    "support": "partner",
+    "currentVersion": "1.0.1",
```

Comment:
```suggestion
    "currentVersion": "1.0.0",
```

Reply:
This will be left as 1.0.1 because I will introduce minor patch changes to the code tomorrow.